modules/order/src/Plugin/Field/FieldFormatter/AdjustmentDefaultFormatter.php

```diff
@@ -0,0 +1,117 @@
+<?php
+
+namespace Drupal\commerce_order\Plugin\Field\FieldFormatter;
+
+use Drupal\commerce_price\NumberFormatterFactoryInterface;
+use Drupal\Core\Entity\EntityTypeManagerInterface;
+use Drupal\Core\Field\FieldDefinitionInterface;
+use Drupal\Core\Field\FieldItemListInterface;
+use Drupal\Core\Field\FormatterBase;
+use Drupal\Core\Plugin\ContainerFactoryPluginInterface;
+use Symfony\Component\DependencyInjection\ContainerInterface;
+
+/**
+ * Plugin implementation of 'commerce_order_adjustment_default_formatter'.
+ *
+ * @FieldFormatter(
+ *   id = "commerce_order_adjustment_default_formatter",
+ *   label = @Translation("Order adjustments"),
+ *   field_types = {
+ *     "commerce_order_adjustment",
+ *   },
+ * )
+ */
+class AdjustmentDefaultFormatter extends FormatterBase implements ContainerFactoryPluginInterface {
```

Comment:
Not sure I see a point in this class. Remember that a list of adjustments does not represent the order total summary (cause that's the line item adjustments + the order adjustments)